#!/usr/bin/env python
PACKAGE = "qm_controllers"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("kp_arm_wbc", double_t, 0, "Kp of arm wbc", 0.0, 0.0, 100)
gen.add("kd_arm_wbc", double_t, 0, "Kd of arm wbc", 0.5, 0.0, 100)

exit(gen.generate(PACKAGE, "weight", "Weight"))
